src/lib/coverage-pool/coverage-pools.js

```diff
@@ -0,0 +1,141 @@
+import { Token } from "../liquidity-rewards/utils"
+import { APYCalculator } from "../helper"
+import { Contract, providers } from "ethers"
+import AssetPool from "./abis/AssetPool.json"
+import KeepToken from "./abis/KeepToken.json"
+import RewardsPool from "./abis/RewardsPool.json"
+import { UniswapExchangeService } from "../exchange-api/exchange-service"
+import BigNumber from "bignumber.js"
+
+const NETWORK = 1
+const INFURA_RPC_URL =
+  "https://mainnet.infura.io/v3/5348113af423473a85e22220ef987de8"
```

Comment:
Could we move it to eg. `lib/web3-config/index.js`?

`lib/web3-config/index.js`
```
export const NETWORK = 1
export const INFURA_RPC_URL =
  "https://mainnet.infura.io/v3/5348113af423473a85e22220ef987de8"
```

We already have the same values in `lib/liquidity-rewards/liquidity-rewards.js`

Reply:
Great idea! Fixed this in 18b8a6ddb5f45d66bcb99f5408675f3ab95f414b